agent/containers/images/visualizers/live-metric-visualizer/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+python3 prom_ds.py &
```

Comment:
Why does this need to be run in the background?

Reply:
That I thought was somewhere between "weird" and "clever" ... maybe both; it just drops the configuration command into the background, and then takes over the parent process with the server that the configuration command is looking for. So once its started, the background command does its thing and goes away.

Again, I think some descriptive comments would help.